src/ddspdrum/module.py

```diff
@@ -411,17 +500,50 @@ def __init__(
         self.noise_module = noise_module
         self.vca = vca
 
+        # Setup parameters for this drum synth
+        self.add_parameter("sustain_duration", sustain_duration, 0, 60, scale=0.5)
+        self.add_parameter("vco_1_ratio", vco_1_ratio, 0, 1)
+
+        # Pitch Envelope
```

Comment:
I'm wondering if we can get rid of this boilerplate using the following approach:
* Each module has a string name, that we define when we create it
* SynthModule double checks it doesn't contain modules with duplicated name
* To address a parameter, you use a tuple: ("module_name", "parameter_name")

Reply:
I removed it https://github.com/turian/ddsp-drum/pull/20 that these are no longer add_parameter.

However, if we stay with the pattern of `connect_parameter` instead of `tuples`, we don't need my change per se.